client/package.json

```diff
@@ -46,5 +46,6 @@
   "moduleNameMapper": {
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
     "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
-  }
+  },
+  "proxy": "http://localhost:3030"
```

Comment:
Maybe we can wait for the backend to be fully active online. And place that URL directly here

Reply:
Server url: https://tripen-server.herokuapp.com

Reply:
Ammended.